### ISSUE TITLE
Org related resources

### ISSUE THREAD

**Issue by mabashian** [ansible/awx#4949]
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY
In the existing application there are number of related lists that can be accessed from the or list:

<img width="531" alt="Screen Shot 2019-10-09 at 9 41 19 AM" src="https://user-images.githubusercontent.com/9889020/66486660-0e196b00-ea79-11e9-9e92-2df8c81b91a3.png">

In list form:

Users - we have this already (Access tab)
Teams - we have this already (Teams tab)
Inventories - we don't have an interface for this (but you could get this information in the future from a search on the inventories list)
Projects - we don't have an interface for this (but you could get this information in the future from a search on the projects list)
Job Templates - we don't have an interface for this (but you could get this information in the future from a search on the templates list.  Query looks like: `or__jobtemplate__project__organization=:id&or__jobtemplate__inventory__organization=:id`)
Admins - we don't have an interface for this (but you could get this information in the future from a search on the org access list).  The existing app interacts with `/api/v2/organizations/1/admins/` for this info.

So the question is, do we need to add these lists into awx-pf.  I'm not keen on removing interfaces without clear reasoning so I _think_ we'll want to add most, if not all of this to ui_next.  @trahman73 @wenottingham ping for thoughts?

**Comment by trahman73** [comment]
@mabashian We display Members and Teams in the current awx-pf list here:

https://tower-mockups.testing.ansible.com/patternfly/orgs/orgs/

I'm not sure how much use the rest of those links got on that view so I'm in favor of going with less and adding later if needed.

**Comment by mabashian** [comment]
OK so I went back through these now that advanced search has been implemented:

Users - we have this already (Access tab)
Teams - we have this already (Teams tab)
Inventories - a user can filter this list down by org ID
Projects - a user can filter this down by org ID
Job Templates - a user can filter this down by doing an org search (name)
Admins - I think that we should add this.  I could not figure out how to get this information effectively via the search.  I'm going to create a new issue to track exposing this information.  Either we'll want to add a related tab for Admins or make sure that the advanced search has a mechanism for filtering the access list down to admins.
